rpc/Cargo.toml

```diff
@@ -37,6 +37,11 @@ ckb-traits = { path = "../traits" }
 ckb-util = { path = "../util" }
 faketime = "0.2.0"
 sentry = "^0.15.4"
+jsonrpc-client-core = "0.5.0"
```

Comment:
Is it only be used in test code, can we move it to `dev-dependencies` section?

Reply:
Got! It's useless dependencies, which I introduced when test. I will remove it.